docs/ref/settings.txt

```diff
@@ -1260,6 +1260,31 @@ Default: ``'``:class:`django.db.models.AutoField`\ ``'``
 Default primary key field type to use for models that don't have a field with
 :attr:`primary_key=True <django.db.models.Field.primary_key>`.
 
+.. admonition:: Migrating auto-created through tables.
+
+    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
+    auto-created through tables for many-to-many relationships.
+
+    The primary key of auto-created through tables cannot currently be updated
```

Comment:
I know it is implicit based on the previous paragraph, but it may be better to be very clear about this:
```suggestion
    The primary key of existing auto-created through tables cannot currently be updated
```